src/test/scala/no/ndla/searchapi/service/search/MultiSearchServiceTest.scala

```diff
@@ -547,7 +541,7 @@ class MultiSearchServiceTest extends IntegrationSuite with TestEnvironment {
     scroll5.results.map(_.id) should be(ids(5))
     scroll6.results.map(_.id) should be(ids(6))
     scroll7.results.map(_.id) should be(ids(7))
-    scroll8.results.map(_.id) should be(List.empty)
```

Comment:
Kanskje legge på en `scroll9` for å fremdeles teste at neste søk er tomt / ikke feiler?